src/researchhub_document/related_models/featured_content_model.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+from typing_extensions import Required
+
+from researchhub_document.related_models.researchhub_unified_document_model import (
+    ResearchhubUnifiedDocument,
+)
+from utils.models import DefaultModel
+
+
+class FeaturedContent(DefaultModel):
+    # Not a foreign key because hub_id=0 is homepage
+    hub_id = models.IntegerField(
```

Comment:
I think this needs to be a foreign key and if it's null then it will be on the homepage

Reply:
I also think in here, we should add something like so:

```
featured = models.BooleanField(...)
```

this way we can do a soft delete and not a hard delete when we remove from featured

Reply:
That's a good suggestion